test/remediation.test.ts

```diff
@@ -0,0 +1,68 @@
+import { test } from 'tap';
+import fs = require('fs');
+
+import plugin = require('../lib');
+import { chdirWorkspaces } from './test-utils';
+const testUtils = require('./test-utils') as any;
+
+test('remediation with constraints.txt', async (t) => {
+  chdirWorkspaces('pip-app');
+  t.teardown(testUtils.activateVirtualenv('pip-app'));
+  const upgrades = {
+    'Django@1.6.1': { upgradeTo: 'Django@2.0.1' },
+    'transitive@1.0.0': { upgradeTo: 'transitive@1.1.1' },
+  };
+  const manifests = {
+    'requirements.txt': fs.readFileSync('requirements.txt', 'utf8'),
+    'constraints.txt': '',
+  };
+
+  const expectedUpdatedManifests = {
```

Comment:
Can the example input and output files please be a separate file in fixtures?  it is much harder to see what changes are being applied when it is all inline and what was initial input?

Reply:
Ok, changed.